Neufund/platform-frontend
# Kk/tx sending modal

How to test it?

Make sure that you have env: `NF_USER_INFO_COMPONENT_ENABLED=1`
On the bottom of the dashboard you will see new button to send TX.

Designs are done in separate PR.
